storage/src/mmr/mem.rs

```diff
@@ -926,13 +903,11 @@ mod tests {
             // Batch-restore the changed leaves to their original values.
             let mut updates = Vec::new();
             for leaf in [0usize, 1, 10, 50, 100, 150, 197, 198] {
-                c_hasher.update(&leaf.to_be_bytes());
-                let element = c_hasher.finalize();
+                hasher.inner().update(&leaf.to_be_bytes());
```

Comment:
nit: wonder if we should wrap these calls? don't love the idea of inner modification this way.

Reply:
In these test cases I'm just using inner() to avoid having to declare another hasher. More generally, inner() allows custom hashers to incorporate additional digest material into the computation, as is done by Bitmap::root() in order to make sure the "partial" chunk is included in the root. So it's not just a test-case hack.

Reply:
https://github.com/commonwarexyz/monorepo/issues/1091